tests/functional/pages/surveys/question.page.js

```diff
@@ -33,6 +33,11 @@ class QuestionPage {
     return browser.click('a[id="top-previous"]')
   }
 
+  next() {
```

Comment:
Yeah, that's fair enough. But in this case, I'm not concerned with routing. I just want to get to the thank you page so the test just keeps on hitting continue until it reaches thank you. I'd argue that having to import > 40 different files just to do that wouldn't help with readability of the test.